packages/back-end/src/scim/groups/groups.router.ts

```diff
@@ -0,0 +1,12 @@
+import { Router } from "express";
```

Comment:
Do we need to include any `group` code in the MVP since they won't be supported?

Reply:
We don't! Just have it here in case I'm able to finish cleaning up the group endpoints in time too. Otherwise I'll remove